src/index.html

```diff
@@ -22,6 +22,413 @@
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+          </div>
+          <div class="card__review-count">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-label">Price:</div>
+          <div class="card__price-amount">$2,199</div>
+        </div>
+
+        <a
+          class="card__product-buy"
+          href="#"
+        >
+          Buy
+        </a>
+      </article>
+
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Product-image"
+          class="card__picture"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+          </div>
+          <div class="card__review-count">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-label">Price:</div>
+          <div class="card__price-amount">$2,199</div>
+        </div>
+
+        <a
+          class="card__product-buy"
+          href="#"
+        >
+          Buy
+        </a>
+      </article>
+
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Product-image"
+          class="card__picture"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+          </div>
+          <div class="card__review-count">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-label">Price:</div>
+          <div class="card__price-amount">$2,199</div>
+        </div>
+
+        <a
+          class="card__product-buy"
+          href="#"
+        >
+          Buy
+        </a>
```

Comment:
The repeated `article` blocks suggest that this should be a list of similar items. Consider using a templating engine or a loop in JavaScript to generate these repeated blocks of code to adhere to the DRY (Don't Repeat Yourself) principle.